codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/swiftintegrations/RegistryConfigIntegration.kt

```diff
@@ -0,0 +1,43 @@
+package software.amazon.smithy.aws.swift.codegen.swiftintegrations
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.swift.codegen.SwiftDelegator
+import software.amazon.smithy.swift.codegen.SwiftSettings
+import software.amazon.smithy.swift.codegen.core.SwiftCodegenContext
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.integration.SwiftIntegration
+
+class RegistryConfigIntegration : SwiftIntegration {
```

Comment:
Writes the registry configuration file into the service client package, so the Swift build tools know where the registry is located.